frontend-new/src/sensitiveData/services/encryptionService/encryptionService.ts

```diff
@@ -0,0 +1,150 @@
+import { EncryptionConfig } from "src/sensitiveData/config/encryptionConfig";
```

Comment:
perhaps rename this file to `encryption.service.ts` and the tests to `encryption.service.test.ts` and the types to `encryption.types.ts` to follow the convention for the other services?

Reply:
now that I see it there is a jumble of these conventions, the auth ones have the `.service.` convention, the others dont...

Reply:
some more tech debt to clean up :)